app/locale/en.coffee

```diff
@@ -186,6 +186,50 @@
     campaign_multiplayer_description: "... in which you code head-to-head against other players."
     campaign_old_multiplayer: "(Deprecated) Old Multiplayer Arenas"
     campaign_old_multiplayer_description: "Relics of a more civilized age. No simulations are run for these older, hero-less multiplayer arenas."
+    
+  code:
+    if: "if" # Keywords
+    else: "else"
+    elif: "elif"
+    while: "while"
+    loop: "loop"
+    for: "for"
+    break: "break"
+    continue: "continue"
+    then: "then"
+    do: "do"
+    end: "end"
+    function: "function"
+    def: "def"
+    self: "self"
+    hero: "hero"
+    this: "this"
+    Date: "Date" # Globals
+    Vector: "Vector"
+    Array: "Array"
+    Function: "Function"
+    Math: "Math"
+    Number: "Number"
+    Object: "Object"
+    RegExp: "RegExp"
+    String: "String"
+    isFinite: "isFinite" # Built-ins
```

Comment:
I think these builtins will actually be covered via the component docs in the `programming.UsesGlobals` Component, so we can cut these from `en.coffee`. (Now, most of the time players don't have that Component, but we can add it to the appropriate Programmaticons as we start to introduce these concepts.) The keywords and globals themselves can stay here.